roles/default/kiali-deploy/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 - debug:
     msg: INSTALLING KIALI
 
+- name: Validate web_schema configuration
+  fail:
+    msg: "Invalid server.web_schema [{{ kiali_vars.server.web_schema }}]! Must be empty, or one of either 'http' or 'https'"
+  when:
+  - kiali_vars.server.web_schema != ''
+  - kiali_vars.server.web_schema != 'https'
+  - kiali_vars.server.web_schema != 'http'
+
```

Comment:
OK, if we are keeping this, can you move this down? I would like this away from the top.. move this down to where we also do some checking of the web_root. So these lines should go on about line 286 (put it after the web_root processing). So put this about here: https://github.com/kiali/kiali-operator/blob/v1.18.1/roles/default/kiali-deploy/tasks/main.yml#L286

Reply:
It's moved down.